AIDriver.lua

```diff
@@ -493,6 +500,49 @@ function AIDriver:driveCourse(dt)
 	end
 end
 
```

Comment:
for now we are still using the old self.triggerHandler:isInTrigger(), which uses the old triggers.lua code, might be a good idea to not depent on that code in the long run